ibis/expr/analysis.py

```diff
@@ -793,3 +793,13 @@ def finder(node):
         )
 
     return g.traverse(finder, nodes, filter=ops.Node)
+
+
+def cumulative_to_window(
+    func: ops.Cumulative, frame: ops.WindowFrame
+) -> ops.WindowFunction:
+    klass = getattr(ops, func.__class__.__name__.replace("Cumulative", ""))
```

Comment:
It looks a bit fragile to me but fine for now.

Reply:
We haven't changed the name of cumulative operations ever ... maybe?